documentation/SA1649.md

```diff
@@ -17,60 +17,34 @@
 
 ## Cause
 
-The file tag within the file header at the top of a C# code file does not match the first type declared in the file. For generics that are defined as `Class1<T>` the name of the file needs to be Class1{T}.cs and this should appear in the header also. Partial classes are ignored.
+The file name of a C# code file does not match the first type declared in the file. For generics that are defined as `Class1<T>` the name of the file needs to be `Class1{T}.cs` or `Class1``1.cs` depending on the `fileNamingConvention` setting. See [Configuration.md](documentation/Configuration.md) for more information.
```

Comment:
:bulb: The link to **Configuration.md** is broken. It needs to be just the following:

```md
[Configuration.md](Configuration.md)
```